### PR TITLE
seperate fio filename and device

### PR DESCRIPTION
**Pull request by wanghenshui**
fio stress filename with device may destroy the disk data,

patch with this

### CODE REVIEW

File: diskplorer.py [hunk 1]
```diff
@@ -121,7 +123,7 @@ def global_section():
             runtime={args.test_step_time_seconds}s
             time_based=1
             ramp_time=5
-            filename={args.device}
+            filename={args.filename}
```
Comment (avikivity) [inline diskplorer.py:126]: The test is designed to work on the raw device and not require the user to create a file (how will they know the size?)

Comment (wanghenshui) [reply diskplorer.py:126]: `filename`'s size could be specific by `size-limit`
eg.

```bash
 sudo ./diskplorer.py /dev/nvme1n1 --filename=/data/tmp/tempfile --result-file koxia.json --size-limit=1G
```

Comment (avikivity) [reply diskplorer.py:126]: Well, it makes the whole thing harder to use, since now the user has to prepare the system.

Comment (wanghenshui) [reply diskplorer.py:126]: if user dont specific `args.filename`, `args.filename` equal with `args.device`

change diskexploer's behavier like that?

Comment (avikivity) [reply diskplorer.py:126]: Well, maybe, but I wasn't able to guess it, and neither will the user.

**Pull request closed without merging**